config/crd/bases/cluster.x-k8s.io_clusters.yaml

```diff
@@ -13,6 +13,14 @@ spec:
     description: Cluster status such as Pending/Provisioning/Provisioned/Deleting/Failed
     name: Phase
     type: string
+  - JSONPath: .status.ready
+    description: Total number of clusters ready
+    name: Ready
+    type: integer
+  - JSONPath: .status.initialized
+    description: Cluster is in initialized state or not(True/False)
+    name: Initialized
+    type: boolean
```

Comment:
These changes for the cluster object and not `kubeadmcontrolplanes`. Also, `Ready` is an integer.

Reply:
Thanks for the feedback @rudoi @wfernandes! I will do the changes in `kubeadm_control_plane_types.go` 

@wfernandes I will change the `Ready` Type to Boolean. Do we need  `Status.ReadyReplicas` as well in the output?

Reply:
@hpandeycodeit For now, I think we can leave it out. 
To me `.ReadyReplicas` provides value if we add the other fields such as `.UnavailableReplicas` and `.Replicas`. I didn't want to clutter the output with all of that information.

Reply:
@detiber ^^^ Regarding your comment about adding ReadyReplicas, etc...
I'm ok adding **all** these fields in, I just didn't want to clutter up the output.

Reply:
I think it's okay to add, and it would be inline with other scalable resources (such as Deployment)